testing/lib/workarea/test_case.rb

```diff
@@ -169,13 +169,26 @@ def reset_s3
       end
     end
 
+    module MorrisDayAndTheTime
+      extend ActiveSupport::Concern
+
+      included do
+        teardown :travel_back
+      end
+
+      def in_three_years
```

Comment:
Why not `next_year` => `1.year.from_now.year`? Is there something special about 3 years?